docs/help-center/using-famedly/chats.mdx

```diff
@@ -273,22 +273,26 @@ Loading the archive may take some time.
   </TabItem>
 </Tabs>
 
-## Check the online status of users
+## Check the status of users
 
-You can see the online status of other users on the profile pictures in the following places:
+You can see the status of other users on the profile pictures in the following places:
 - in the chats list
 - in the contact list (when opening a new chat)
 - on the title bar of 1:1 chats
 - in group chats above the single messages from other users
-- in the member list of group chats  
+- in the member list of group chats 
 
-### Which symbol shows which online status?
+### Which symbol shows which status?
 
-**Online** &rarr; Green dot on the profile picture   
-**Busy** &rarr; Red dot with a white line on the profile picture  
-**Offline** &rarr; Grey dot on the profile picture  
+**Active** &rarr; Green dot on the profile picture
+**Away** &rarr; Grey dot on the profile picture  
 
-Also you can see the online status as text in the title bar of 1:1 chats.
+Also you can see the status as text in the title bar of 1:1 chats. In previous versions "online" has been displayed instead of "active".
+
+### What does this status mean?
+
+**Active** &rarr; The user has the app open and visible on a screen on a mobile device **or** the web version of the app is open in a browser tab
+**Away** &rarr; After 10 minutes when the user has swiped away the app on mobile or closed the browser tab, the status becomes automatically **away**.
```

Comment:
Has closed the app on mobile? Why do we say swiped away?
'...the status changes to: away' sounds better for me.